lib/root-dir.js

```diff
@@ -1,8 +0,0 @@
-const {app, process: remoteProcess} = require('electron').remote
-const minimist = require('minimist')
-const fs = require('fs')
-
-const argv = minimist(remoteProcess.argv.slice(2))
-const rootDir = argv.data || `${app.getPath('downloads')}/dat`
```

Comment:
can we add this back? this is required so you can run two instances of dat-desktop on the same machine, to verify they're able to replicate with each other properly